### DISCUSSION TITLE
optimized permeability are not getting committed to compound property

### DISCUSSION THREAD

**Discussion by bhim1992** [Open-Systems-Pharmacology/Forum#1786] [Questions & Problems] [answered]
The optimized permeability are not getting committed to compound property, as on each new simulation it needs manual feeding. However on cloning it's working. How to commit optimised/fitted permeability across organs to compound block that can be saved in templet compound for further use in plugging with another drug model for DDI evaluation? 

Thank you in advance for your support. 
Regards, 
Bhim

**Comment by Yuri05** [comment]
1. In your compound, create a new alternative for permeability
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/a3defa3a-00cc-41ab-8e37-2d62d49c9aa5)

2. Make sure to use this permeability alternative when creating/updating a simulation:
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/57145054-364d-48dc-94e5-2a7ce5233f89)

3. Update your optimized permeability value in used simulation(s) from the parameter identification via "Transfer to simulation" button

4. Use the "Commit to Building block" functionality to transfer the optimized permeability value back into the compound
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/34aebfc7-df77-4a1f-be9a-66559ac5b5c5)

**Comment by bhim1992** [comment]
Dear Yuri, Thank you for your quick guide. As you suggested, it works well to optimize specific intestinal permeability. However, my query is for the permeability optimized specifically for particular organs as shown in the picture.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/127585790/a358e6a8-d033-463c-8a33-88e840d01440)

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/127585790/6bbd1d41-a44e-44aa-aef2-29f264ba4ee5)

After committing to building blocks, on adding new simulations, the permeability changes for the brain and fat from parameter identification doesn't update, It reflects again the same previous values.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/127585790/1835a074-f1a0-4858-bd23-7549f94c36d7)

The optimized values for permeability to the organ (here brain and fat) are not committed to building blocks, so in the new simulation, again, I need to add fitted values manually. 
Need ideas to commit to this permeability value.


Thank you.

**Reply by Yuri05** [reply, marked as answer]
OK, I see. This indeed is not possible at the moment.
The problem is, that organ permeabilities are defined on the simulation level only (because they depend both on compound and individual properties) and thus cannot be transferred back e.g. into the compound.

There is an ongoing discussion/feature request on this topic (https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2035) and this extension will be probably implemented in PK-Sim 13.

**Reply by bhim1992** [reply]
Thank you.
